wordpress-mobile/WordPress-iOS
# Resolve crash in WPRichContentView with content set to empty string.

Fixes #10330.

### To test

- Review issue description, including offending commit & Fabric crash disposition.
- Verify branch builds & tests pass.
- Revert commit `8bc8244` and verify that `WPRichContentViewTests` fails by crashing with the same disposition.
- Consider manual testing via the editor.
